README.md

```diff
@@ -1,4 +1,4 @@
-# Logbook
+# Logbook: an extensible Java library for HTTP request and response logging

```

Comment:
I wouldn't put this in the headline. Now it breaks into multiple lines. Either a normal-text *sub-headline* or we put it as the first sentence to the description.